.storybook/webpack.config.js

```diff
@@ -6,7 +6,7 @@ const webpack = require.main.require('webpack');
 module.exports = storybookBaseConfig => {
 	// remove Uglification plugin to improve build time in CI
 	const uglifyIndex = storybookBaseConfig.plugins.findIndex(
-		element => element instanceof webpack.optimize.UglifyJsPlugin
+		element => element instanceof webpack.optimize.UglifyJsPlugin,
 	);
```

Comment:
Be carefull of trailing coma. The prod's node don't support trailing coma at the moment

Reply:
LTS is supporting it